File: logica/listaDeReforco/C/e9.c
#include <stdio.h>

int main(){
    int op;
    char ops[5][40] = {"depósito", "saque", "extrato", "transferencia", "Encerrando operações"};
    printf("---------------\n Banco IFSP\n ---------------\n 1. depósito\n 2. saque\n 3. extrato \n 4. transferencia \n---------------\n 5 = encerrar\n---------------\n Sua escolha:\n");
    scanf("%d", &op);
    if(op == 5){
        printf("%s", ops[op-1]);
    }else{
        printf("Operacao de %s", ops[op-1]);
    }
}

File: logica/listaDeReforco/Python/e9.py
print("---------------\n Banco IFSP\n ---------------\n 1. depósito\n 2. saque\n 3. extrato \n 4. transferencia \n---------------\n E = encerrar\n---------------\n Sua escolha: ")
escolha = input()
resp = ["depósito", "saque", "extrato", "transferencia", "Encerrando operações"]
if escolha == "1" or escolha =="2" or escolha =="3" or escolha =="4":
    print(f"Operação de {resp[int(escolha)-1]}")
else:
    escolha.capitalize()
    if escolha != "E":
        print("Encerrando operações")